hooks/command

```diff
@@ -200,7 +203,10 @@ else
     else
       sleep "$job_status_retry_interval_sec"
       if [[ $timeout -gt 0 ]]; then
-        (( counter -= job_status_retry_interval_sec ))
+        if ! (( counter -= job_status_retry_interval_sec )); then
+          echo "Warning: never successfully got pod status"
+          exit 42
+        fi
```

Comment:
```suggestion
        (( counter -= job_status_retry_interval_sec )) || {
          echo "Warning: never successfully got pod status"
          exit 42
        }
```
nit: we can use this format instead only for legibility reasons to avoid the extra if statement nesting (same for the above one)